### PR TITLE
feat: new ipfs-css with new failed pin icon 

### PR DESCRIPTION
**Pull request by hacdias**
Adds the new failed cloud [icon](https://github.com/ipfs/ipfs-webui/pull/1919#issuecomment-1205950576) for the failed pins. This had to rebuild the icons, which changed all of them due to certain dependencies being updated in the meanwhile.

### CODE REVIEW

File: src/pins/PinsStatuses.js [hunk 1]
```diff
@@ -29,7 +30,9 @@ const Pin = (t, pin, status = Status.Pending, onDismiss) => {
   }
 
   return (<li className="flex w-100 bb b--light-gray items-center f6 charcoal" key={cid}>
-    <GlyphPinCloud className={cloudClass} style={{ width: '36px' }} alt={alt} title={alt} />
+    {status === Status.Failed
+      ? <GlyphPinCloudFailed className={cloudClass} style={{ width: '36px' }} alt={alt} title={alt} />
+      : <GlyphPinCloud className={cloudClass} style={{ width: '36px' }} alt={alt} title={alt} />}
```
Comment (SgtPooki) [inline src/pins/PinsStatuses.js:35]: We should encapsulate this logic within GlyphPinCloud if we can

File: src/icons/GlyphPinCloudFailed.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+
+function SvgGlyphPinCloudFailed (props) {
```
Comment (SgtPooki) [inline src/icons/GlyphPinCloudFailed.js:3]: Can we create a new component that contains the `status === 'Failed' ? GlyphPinCloud : GlyphPinCloudFailed` logic? Maybe in `src/icons/HOCs/GlyphPinCloud.js` ?

**Review by SgtPooki — APPROVED** [review]

**Comment by SgtPooki** [comment]
Why have the github actions not run against this PR?

**Comment by hacdias** [comment]
Merged on feat/track-pins. Please check PR #1919.

**Pull request merged**